docs/usage.md

```diff
@@ -277,33 +321,52 @@ The time series stack output can be enabled using:
 
 ### Visualization
 
-The workflow provides two types of results visualization and aggregation. The fine grained mosaic visualization contains all time series analyses results for all tiles in the original resolution. Pyramid visualizations present a broad overview of the same data but at a lower resolution. Both visualizations can be enabled or disabled using the parameters `mosaic_visualization` and `pyramid_visualization`. By default, both visualization methods are enabled. Note that the mosaic visualization is required to be enabled when using the `test` and `test_full` profiles to allow the pipeline to check the correctness of its results (this is the default behavior, make sure to not disable mosaic when using test profiles) .
+The workflow provides two types of results visualization and aggregation.
+The fine grained mosaic visualization contains all time series analyses results for all tiles in the original resolution.
+Pyramid visualizations present a broad overview of the same data but at a lower resolution.
+Both visualizations can be enabled or disabled using the parameters `--mosaic_visualization` and `--pyramid_visualization`.
+By default, both visualization methods are enabled.
+Note that the mosaic visualization is required to be enabled when using the `test` and `test_full` profiles to allow the pipeline to check the correctness of its results.
 
 The visualizations can be enabled using:
 
 ```bash
-mosaic_visualization  = '[boolean]'
-pyramid_visualization = '[boolean]'
+--mosaic_visualization  = '[boolean]'
+--pyramid_visualization = '[boolean]'
```

Comment:
```suggestion
mosaic_visualization  = 'true'
pyramid_visualization = 'true'
```
maybe also add a `title=paramters.yml` to the code block and switch the language. for reproducibility I would discourage cli paramter input and nudge people to use a paramter file instead.

Reply:
I would need to change that for every parameter in usage.md 
Previously I was told to use cli parameter input.
Is there a guideline for that?